R/prepare_data.R

```diff
@@ -182,6 +178,17 @@ prepare_data <- function(data) {
                        "Lichamelijke_klachten", "Plezierig", "Onplezierig",
                        "Slapen")
   
+  # If dataset has many missings on particular items (>50%), then stop
+  for(var in circle_vars_nms) {
+    
+    if( ( sum(is.nan( data[ , var] ) ) / nrow(data) ) > 0.50  ) {
```

Comment:
ZIjn op dit punt de non-responses al uit data gehaald? 

Want anders lijkt me deze vele malen strenger dan `Too many non-responses (>90%)` en is het raar dat we er daar al niet uitknallen. 

Ook zijn alle vragen in principe verplicht om in de vullen, dus verwacht dat deze vooral aangaat als er een categorie geen vraag is geselecteerd?

Reply:
Mogelijk willen losse error voor als er helemaal geen invulling is?